src/components/grid/grid.test.js

```diff
@@ -31,267 +32,167 @@ const viewportSize = (sizeOfViewport) => {
+      (itemNumber, row, col, expectedWidth) => {
+        CypressMountWithProviders(<GridLayoutExample />);
 
         viewportSize("large");
-        if (index === 1) {
-          gridPod(index).should("have.css", "grid-row", "auto / auto");
-          gridPod(index).should("have.css", "grid-column", "auto / auto");
-          gridPod(index)
-            .invoke("css", "width")
-            .then(parseFloat)
-            .then(($el) => {
-              expect($el).to.be.gte(podWidth);
-              expect($el).to.be.lt(podWidth + 1);
-            });
-        } else {
-          gridPod(index).should("have.css", "grid-row", row);
-          gridPod(index).should("have.css", "grid-column", col);
-          gridPod(index)
-            .invoke("css", "width")
-            .then(parseFloat)
-            .then(($el) => {
-              expect($el).to.be.gte(podWidth);
-              expect($el).to.be.lt(podWidth + 1);
-            });
-        }
+        gridItem(itemNumber - 1).should("have.css", "grid-row", row);
+        gridItem(itemNumber - 1).should("have.css", "grid-column", col);
+        gridItem(itemNumber - 1)
+          .invoke("css", "width")
+          .then(parseFloat)
+          .then((width) =>
+            expect(width).to.be.within(expectedWidth, expectedWidth + 1)
+          );
       }
     );
 
     it.each([
-      [1, "1 /1", "1 / 13", 1826],
+      [1, "auto / auto", "1 / 13", 1826],
       [2, "2 / 3", "1 / 6", 737],
       [3, "4 / 5", "7 / 13", 893],
     ])(
-      "should render Grid viewport to extra large and verify pod width and height",
-      (index, row, col, podWidth) => {
-        CypressMountWithProviders(<GridComponentSized />);
+      "when viewport size is extra large, grid item %s should have correct grid-row, grid-column and width",
+      (itemNumber, row, col, expectedWidth) => {
+        CypressMountWithProviders(<GridLayoutExample />);
 
         viewportSize("extra large");
-        if (index === 1) {
-          gridPod(index).should("have.css", "grid-row", "auto / auto");
-          gridPod(index).should("have.css", "grid-column", "auto / auto");
-          gridPod(index)
-            .invoke("css", "width")
-            .then(parseFloat)
-            .then(($el) => {
-              expect($el).to.be.gte(podWidth);
-              expect($el).to.be.lt(podWidth + 1);
-            });
-        } else {
-          gridPod(index).should("have.css", "grid-row", row);
-          gridPod(index).should("have.css", "grid-column", col);
-          gridPod(index)
-            .invoke("css", "width")
-            .then(parseFloat)
-            .then(($el) => {
-              expect($el).to.be.gte(podWidth);
-              expect($el).to.be.lt(podWidth + 1);
-            });
-        }
+        gridItem(itemNumber - 1).should("have.css", "grid-row", row);
+        gridItem(itemNumber - 1).should("have.css", "grid-column", col);
+        gridItem(itemNumber - 1)
```

Comment:
I would rather not use `invoke` here, just use the `jQuery` method using 

`gridItem(itemNumber - 1).then(($el) => {
     expect($el.css("width"))...
})`